### ISSUE TITLE
Accents

### ISSUE THREAD

**Issue by merisjames** [ualbertalib/discovery#866]
Passed on from Denis...
It seems the database treats accented letters differently from non-accented ones. For example, if you search for "editions du ble" you will find 4 titles whereas if you add the accent on the two 'e's, as in "éditions du blé", you will get 170 results. I think it is preferable not to oblige users to include accents. Thank you for sending this along to those still working on Blacklight.

**Comment by redlibrarian** [comment]
Hm. Standard Solr behaviour is not to care about accents, and databases aren't indexed differently than anything else. I'll have to look into this.

**Comment by redlibrarian** [comment]
So, searching with and without diacritics seemed to only be partially working. Following this blog post,  http://yfrankfeng.blogspot.ca/2012/08/search-diacritics-in-solr.html, I modified the schema.xml to use the solr.ASCIIFoldingFilterFactory on all appropriate fields. In the symphony_test_set, before making this change I got:

étude - 56 results
etude - 50 results

After the change I get

étude - 56 results
etude - 56 results

Which is the expected behaviour. I can't integrate this right now as we are staying away from making indexing changes, and this will require a reindex of data to fix. It's currently parked in the "diacritics" branch, and can be merged when we're ready to do a reindex.
